pkg/controller/manager/manager_controller.go

```diff
@@ -404,19 +404,20 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 			}
 			trustedSecretNames = append(trustedSecretNames, render.ComplianceServerCertSecret)
 		}
+	}
 
-		// Fetch the Authentication spec. If present, we use to configure user authentication.
-		authenticationCR, err = utils.GetAuthentication(ctx, r.client)
-		if err != nil && !errors.IsNotFound(err) {
-			r.status.SetDegraded(operatorv1.ResourceReadError, "Error while fetching Authentication", err, logc)
-			return reconcile.Result{}, err
-		}
-		if authenticationCR != nil && authenticationCR.Status.State != operatorv1.TigeraStatusReady {
-			r.status.SetDegraded(operatorv1.ResourceNotReady, fmt.Sprintf("Authentication is not ready authenticationCR status: %s", authenticationCR.Status.State), nil, logc)
-			return reconcile.Result{}, nil
-		} else if authenticationCR != nil {
-			trustedSecretNames = append(trustedSecretNames, render.DexTLSSecretName)
-		}
+	var authenticationCR *operatorv1.Authentication
```

Comment:
I'm not sure this is the right way to do this - I think we will need to namespace this CR, no?

Otherwise, all tenants will share the same Authentication configuration?

Reply:
I was thinking the same thing. At the moment, I was simply trying to get the environment populated for a single tenant.

Reply:
Talked offline. It seems that Cloud uses a TigeraOIDC and does not enable it per each tenant. ManagerURL is used to configure Kibana and IssuerURL and Groups claims have the same value across tenants.

Reply:
I think we should add validation so that when running in multi-tenant mode, we reject an Authentication resource that isn't supported (e.g., Dex mode and other unsupported fields in multi-tenant)